docs/run/configure-error-mitigation.mdx

```diff
@@ -42,6 +42,14 @@ expectation values.
 
 <Admonition type="caution" title="Important">
 To ensure faster and more efficient results, as of 1 March 2024, circuits and observables need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits and observables) before being submitted to the Qiskit Runtime primitives.  See the [transpilation documentation](../transpile) for instructions to transform circuits.  Due to this change, the primitives will no longer perform layout or routing operations.  Consequently, transpilation options referring to those tasks will no longer have any effect. Users can still request that the primitives do no optimization of input circuits by specifying `optimization_level=0`.
+
+
+**Exception**: The Q-CTRL primitives continue to support abstract circuits, as in the following code: 
```

Comment:
I think we got feedback in the past that we shouldn't bold as much because it looks similar to an `h3`. Maybe use italics instead?

Reply:
Happy to use italics here. However, I'm curious about this feedback, especially as we use bold often in the docs (usually as part of a block of text, or with a colon following it) and wonder if in those contexts it would not be mistaken for h3, and can be left alone?